test/server.cpp

```diff
@@ -322,6 +321,19 @@ TEST_F(ServerTest, 404)
     EXPECT_EQ(404, zfs1_->GET(url)->status) << "url: " << url;
 }
 
+const char* urls400[] = {
+  "/ROOT/search",
+  "/ROOT/search?content=zimfile",
+  "/ROOT/search?pattern",
+};
+
+
+TEST_F(ServerTest, 400)
+{
+  for (const char* url: urls400 )
+    EXPECT_EQ(400, zfs1_->GET(url)->status) << "url: " << url;
```

Comment:
Please add braces to the body of the loop

Reply:
Done in a new commit (which add braces around several for loops)